operator/main.py

```diff
@@ -899,6 +919,12 @@ def deploy_csi_pods(core_v1_client):
 
     if api_instance.major > "1" or api_instance.major == "1" and \
        api_instance.minor >= "22":
+
+        if csi_driver_object_api_version() is not None and \
+           csi_driver_object_api_version() == "v1beta1":
```

Comment:
better check for our intention, like we want `v1` in this k8s range, so it'll be `!= "v1"`

Reply:
Yeah maybe because it can be anything other than `v1beta1` as well. Changed it.